### DISCUSSION TITLE
How  to get maya scene path and version from _FinishWriting()

### DISCUSSION THREAD

**Discussion by koen-v** [Autodesk/maya-usd#1303] [Q&A] [answered]
Hello all, 

I am adding some meta data writing to the maya usd writer ( I realize I could do that in a python chaser, but I am customizing this code already anyway, so might as well do it here.

Two of the things I would like to write to the usd file are the maya version that was used and the path of the scene file that was exported from.

from writeJob.cpp, UsdMaya_WriteJob::_FinishWriting() is there an easy way to get both of these?

thanks,
Koen

**Comment by ysiewappl** [comment, marked as answer]
@koen-v Not sure I understand why where it happens (i.e. in `UsdMaya_WriteJob::_FinishWriting()` is important, but couldn't you rely on `MFileIO::currentFile` and `MGlobal::apiVersion`(and the other Maya version-related functions in `MGlobal`) to get this information?

**Reply by koen-v** [reply]
Yes, I got that from one of our internal developers as well, thansk! Should have closed this. The problem is mostly that I  am not familiar with the maya api, but dove in to the plugin to fix and add some things specific to us anyway, need to learn the api I guess.

Thanks!
